examples/chat_playground/server.py

```diff
@@ -6,7 +6,7 @@
 
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
-from langchain.chat_models import ChatAnthropic
+from langchain_anthropic.chat_models import ChatAnthropic
```

Comment:
thank you!

Reply:
changes to the poetry and pyproject files have been rolled back.